src/ResourceManagement/BatchAI/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,24 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// </auto-generated>
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure BatchAI Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure BatchAI management APIs.")]
+
+[assembly: AssemblyVersion("1.0.0.60")]
+[assembly: AssemblyFileVersion("1.3.1.0")]
```

Comment:
>1.3.1.0 [](start = 32, length = 7)

Same comment - please change the version to 1.3.0 (I assume we are not going to publish this package only, right?)